tests/App/Jobs/FetchCollectionFloorPriceTest.php

```diff
@@ -8,9 +8,12 @@
 use App\Models\Token;
 use App\Support\Facades\Mnemonic;
 use Carbon\Carbon;
+use Illuminate\Support\Facades\Config;
 use Illuminate\Support\Facades\Http;
 
 it('should fetch nft collection floor price', function () {
+    Config::set('dashbrd.web3_providers.'.FetchCollectionFloorPrice::class, 'mnemonic');
```

Comment:
We should have a test for Moralis too especially if we now default to it. It doesn't look like we have one yet.

Reply:
@oXtxNt9U no longer neccesary, updated the test to use opensea the new default